### DISCUSSION TITLE
Add {archive_serial_number} placeholder for PAPERLESS_FILENAME_FORMAT

### DISCUSSION THREAD

**Discussion by ghost** [jonaswinkler/paperless-ng#519] [Archived FRs]
The ASN concept explained in the docs is very compelling, but I'd like to be able to export my files with the ASN in the filename, should I ever have to migrate away from paperless in the future.

I know the ASN would be on the scanned documents too, but usually handwritten and not searchable/machine-readable.

I'd prefer the placeholder to be "none" when no ASN is assigned, but whatever works best.

**Comment by jonaswinkler** [comment]
Added! I used `{asn}` to keep it short.
